.github/workflows/ci.yml

```diff
@@ -126,7 +126,7 @@ jobs:
 
   test:
     name: Test UE ${{ matrix.unreal }}
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-4-cores-arm64
```

Comment:
@bitsandfoxes should we make this a matrix instead? So it runs on both 4-cores and 4-cores-arm64?

Reply:
Oh actually that won't work because there are not arm64 docker images for UE?

> WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

Reply:
There's no official arm64 docker images for UE so it won't work (for now at least). However we may utilize our new arm64-runner for the future CI pipeline improvements (#575) and proper support of this arch (#473)